### PR TITLE
S4 create page

### PR DESCRIPTION
**Pull request by f-gate**
files to review:
index.scss
pokestyles.scss
imports/variables.scss
gulpile.js
index.php
login.php

made the index.php and index.scss moved the variables to an imports folder and imported into both scss files, changed gulpfile to accommodate new scss files.

### CODE REVIEW

File: scss/index.scss [hunk 1]
```diff
@@ -0,0 +1,126 @@
+@import "imports/variables";
+
+@font-face {
+  font-family: Ketchum;
+  src: url('../fonts/Ketchum.otf');
+}
+
+body {
```
Comment (patrickkenyon) [inline scss/index.scss]: box-sizing needs to go on all elements e.g 

```*``` {
box-sizing: border-box
}

File: scss/index.scss [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+}
+
+.poke-container {
+  width: 100%;
+  height: 150px;
+  outline-style: solid;
+  outline-offset: -2px;
+}
+
+.info {
```
Comment (patrickkenyon) [inline scss/index.scss:42]: .info and .picture-holder need to add to 100% currently add to 95%

File: scss/index.scss [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+h2 {
+  margin-bottom: 0;
+  margin-top: 2%;
+}
+
+.poke-container {
+  width: 100%;
+  height: 150px;
+  outline-style: solid;
```
Comment (patrickkenyon) [inline scss/index.scss]: using outlines is causing borders to double up also outline-offset -2px is unusual way to achieve a border.  Consider 
border-left:1px solid #222222
border-bottom:1px solid #222222
border-right:1px solid #222222

File: scss/index.scss [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+  height: 100%;
+  float: left;
+  text-align: center;
+  overflow: auto;
+}
+
+.icon-container {
+  float: left;
+  height: 40%;
+  padding: 3% 12% 0 12%;
```
Comment (Benson-J) [inline scss/index.scss:85]: Could this be achieved with a margin: auto?

File: scss/pokestyles.scss [hunk 5]
```diff
@@ -0,0 +1,83 @@
+@import "imports/variables";
```
Comment (Benson-J) [inline scss/pokestyles.scss:1]: Does not appear to be used on list page, but has been requested for review.

File: scss/index.scss (hunk 4 above)
Comment (f-gate) [reply scss/index.scss:85]: no

File: scss/index.scss (hunk 1 above)
Comment (f-gate) [reply scss/index.scss]: done in  e919a8b

File: scss/index.scss (hunk 2 above)
Comment (f-gate) [reply scss/index.scss:42]: done in 
e80e691

File: scss/index.scss (hunk 3 above)
Comment (f-gate) [reply scss/index.scss]: done in 
3aa992d with border: 1px solid #222222;

**Review by Benson-J — APPROVED** [review]

**Pull request merged**